### ISSUE TITLE
RASA_NLU : Error while training the chatbot

### ISSUE THREAD

**Issue by sbadrinath** [RasaHQ/rasa_nlu#1458]
**Rasa NLU version**: 0.13.6

**Operating system** (windows, osx, ...): windows 8.1 Pro

**Content of model configuration file**:
```config.json

{
  "pipeline": "spacy_sklearn",
  "path" : "./",
  "data" : "./nlu_data/gstfaq-data.json"
}

```

**Issue**:

Python 3.7.0
pip version 10.0.1

I am a beginner. I installed Anaconda and running the below command to train rasa_nlu from Anaconda command prompt as an Administrator and I am getting the error. I am attaching the gstfaq-data.json file
Note : I have renamed gstfaq-data.json to gstfaq-data.txt for attachment purpose

[gstfaq-data.txt](https://github.com/RasaHQ/rasa_nlu/files/2463981/gstfaq-data.txt)

(base) c:\Badri\rasa_nlu\BOTProject\DBABOT>python -m rasa_nlu.train -c config.json --data nlu_data/gstfaq-data.json
Fitting 2 folds for each of 6 candidates, totalling 12 fits
C:\Users\bshanmug\AppData\Local\Continuum\anaconda3\lib\site-packages\sklearn\metrics\classification.py:1135: 
UndefinedMetricWarning: F-score is ill-defined and being set to 0.0 in labels with no predicted samples.
  'precision', 'predicted', average, warn_for)

**Comment by sbadrinath** [comment]
Below Article was helpful
https://github.com/RasaHQ/rasa_nlu/issues/288

Per, increased the examples which resolved the issue.

Before
=====

2018-10-10 15:37:01 INFO     rasa_nlu.training_data.training_data  - Training data stats:
        - intent examples: 28 (6 distinct intents)
        - Found intents: 'goodbye', 'gst-info', 'greet', 'gst-query', 'intro', 'affirm'
        - entity examples: 9 (4 distinct entities)
        - found entities: 'gst-query-type', 'gst-info-value', 'gst-query-value', 'gst-info-type'

After
====		
2018-10-10 16:16:46 INFO     rasa_nlu.training_data.training_data  - Training data stats:
        - intent examples: 34 (6 distinct intents)
        - Found intents: 'affirm', 'intro', 'gstinfo', 'gstquery', 'greet', 'goodbye'
        - entity examples: 10 (4 distinct entities)
        - found entities: 'gstinfo-type', 'gstquery-type', 'gstinfo-value', 'gstquery-value'
